CMSgov/easi-app
# [EASI-3851] Contracts system profile

# EASI-3851


## Changes and Description

`Vendors` and `Contract services or functions` data is not yet connected to CEDAR and left out of the scope of this work

- Updated `GetSystemProfileQuery` to include contracts
- (BE Change - Clay) - Fixed contractNumber returning incorrect data
- Added Contacts route, sidenav, and component
- Updated date utils to allow null dates
- Unit test



## How to test this change

- Make sure LD flag is toggled to allow hidden system profile work to be rendered
- Click `Contracts` from system prolfile side nav
- Verify data render
- Verify empty states

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
